lib/msal-electron-proof-of-concept/src/AppConfig/ServerResponse/ServerResponse.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as url from 'url';
+
+type QueryParameters = {
+    code?: string;
+    error?: string;
+    error_description?: string;
+};
+
+export abstract class ServerResponse {
```

Comment:
This class already has a sub-class (AuthCodeResponse). The next step will involve another sub-class for Token request responses (TokenResponse). We'll add the 'token' and  'refresh_token' elements then.